metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Installs and configures rsyslog'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '2.1.1'
+version           '2.1.2'
```

Comment:
The version should not be rev'd here.  I'll handle the version bumps during the release phase.